docs/cli-reference/dfx-canister.md

```diff
@@ -389,7 +389,7 @@ dfx canister install --all
 
 #### Installing a specific canister
 
-You can also use the `dfx canister install` command to deploy a specific canister instead of all of the canisters in your project. For example, if you have a project with a `hello_world` canister and a `hello_world_assets` canister but only want to deploy the `hello_world` canister, you can deploy just that the canister by running the following command:
+You can also use the `dfx canister install` command to deploy a specific canister instead of all of the canisters in your project. For example, if you have a project with a `hello_world` canister and a `hello_world_frontend` canister but only want to deploy the `hello_world` canister, you can deploy just that the canister by running the following command:
 
 ``` bash
 dfx canister install hello_world
```

Comment:
```suggestion
You can also use the `dfx canister install` command to deploy a specific canister instead of all of the canisters in your project. For example, if you have a project with a `hello_world_backend` canister and a `hello_world_frontend` canister but only want to deploy the `hello_world_backend` canister, you can deploy just that the canister by running the following command:

``` bash
dfx canister install hello_world_backend
```

Reply:
I agree. I had to do this manually in the new branch